toolkit/tools/imagecustomizer/container/Dockerfile.mic-container

```diff
@@ -2,5 +2,22 @@ FROM mcr.microsoft.com/cbl-mariner/base/core:2.0
 RUN tdnf update -y && \
    tdnf install -y qemu-img rpm coreutils util-linux systemd openssl \
       sed createrepo_c squashfs-tools cdrkit parted e2fsprogs dosfstools \
-      xfsprogs zstd veritysetup
+      xfsprogs zstd veritysetup wget tar gnupg
+
+# Set environment variables for ORAS installation.
+ARG ARCH="amd64"
+ARG VERSION="1.1.0"
+
+# Install ORAS CLI tool.
+RUN curl -LO "https://github.com/oras-project/oras/releases/download/v${VERSION}/oras_${VERSION}_linux_${ARCH}.tar.gz" && \
+    curl -LO "https://github.com/oras-project/oras/releases/download/v${VERSION}/oras_${VERSION}_linux_${ARCH}.tar.gz.asc" && \
+    curl -sSL https://raw.githubusercontent.com/oras-project/oras/v${VERSION}/KEYS | gpg --import && \
+    gpg --verify oras_${VERSION}_linux_${ARCH}.tar.gz.asc oras_${VERSION}_linux_${ARCH}.tar.gz && \
```

Comment:
Remove the call to `gpg`. A signature check doesn't achieve anything if you are downloading the public key over the same channel as the files themselves.